app/jobs/organization_event_job.rb

```diff
@@ -5,21 +5,31 @@ class OrganizationEventJob < ApplicationJob
   queue_as :github_event
 
   # rubocop:disable Metrics/AbcSize
+  # rubocop:disable Metrics/MethodLength
   def perform(payload_body)
     return true unless payload_body.dig("action") == "member_removed"
 
     github_user_id = payload_body.dig("membership", "user", "id")
     github_organization_id = payload_body.dig("organization", "id")
-    @organization = Organization.find_by(github_id: github_organization_id)
+    organizations = Organization.where(github_id: github_organization_id)
 
-    return false if @organization.blank?
-    return false if @organization.users.count == 1
+    return false if organizations.empty?
 
-    @user = @organization.users.find_by(uid: github_user_id)
+    failed_removals = organizations.reject do |org|
+      user = org.users.find_by(uid: github_user_id)
```

Comment:
I am not sure how many orgs we will generally have or if there is a way to avoid this N+1. It seems like it will be negligible and in a job... so... not a big deal?

Reply:
I tried fixing this by using an includes where I grab the organizations above:

`organizations = Organization.where(github_id: github_organization_id).includes(:users)`

However, I end up with an error from the bullet gem:

```
     Bullet::Notification::UnoptimizedQueryError:
       user: jessrudder
        
       AVOID eager loading detected
         Organization => [:users]
         Remove from your finder: :includes => [:users]
       Call stack
         /Users/jessrudder/github/classroom/spec/support/bullet.rb:16:in `block (2 levels) in <top (required)>'
         bin/rspec:16:in `load'
         bin/rspec:16:in `<main>'
```

If an `includes` would be the right thing to do here to avoid the N+1, I can whitelist this for the bullet gem so it doesn't unfairly fail the test. If not, is there another way to avoid the N+1?

Reply:
Just to confirm I think this is probably okay as an N+1, I don't think we will have too many of these.